tests/outputs/cv/scopatz.tex

```diff
@@ -327,29 +327,29 @@
     {Professor}
     {Aug 2017
     }
-    { 
+    {
         \resitem{This course is an introduction to nuclear physics.}
-      
-      
+
+
         \resitem{
             \href{https://drive.google.com/open?id=0BxUpd34yizZreDBCMEJNY2FUbnc}{Syllabus}.
         }
-      
+
     }
 
 \ressubheadingitems{EMCH 558/758: Reactor Power Systems}
     {University of South Carolina}
     {Professor}
     {Jan 2017
     }
-    { 
+    {
         \resitem{This course covers conventional reactors.}
-      
-      
+
+
         \resitem{
-            \href{https://docs.google.com/document/d/1uMAx_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
+            \href{https://docs.google.com/document/d/1uMAx\_KFZK9ugYyF6wWtLLWgITVhaTBkAf8-PxiboYdM/edit?usp=sharing}{Syllabus}.
```

Comment:
Did you mean to put in a `\` before the `_`?

Reply:
Yep, absolutely.  `_` on their own aren't renderable.

Reply:
Yes but `\href` doesn't render the URL, right? It should give a link named `Syllabus` and the link is the URL.

Reply:
Maybe it is just `#` that fail to render without and escape then.